jetty-core/jetty-deploy/src/main/java/org/eclipse/jetty/deploy/DeploymentScanner.java

```diff
@@ -580,19 +580,25 @@ void resetAppState(String name)
         app.resetStates();
     }
 
-    @ManagedOperation(value = "Scan the monitored directories", impact = "ACTION")
+    @ManagedOperation(value = "Scan the webapps directories", impact = "ACTION")
     public void scan()
     {
-        LOG.info("Performing scan of monitored directories: {}",
-            monitoredDirs.stream()
+        LOG.info("Performing scan of webapps directories: {}",
+            webappDirs.stream()
                 .map(Path::toUri)
                 .map(URI::toASCIIString)
                 .collect(Collectors.joining(", ", "[", "]"))
         );
         scanner.nudge();
     }
 
-    private List<DeployAction> buildActionList(List<PathsApp> changedApps)
+    @Override
+    public String toString()
```

Comment:
Move it as the last method of the class, where it was before.

Reply:
Huh?  I'll do it, but I didn't catch that nuance in the merge from `jetty-12.1.x`